src/connectors/aws_inventory.py

```diff
@@ -192,22 +192,26 @@ def ingest(table_name, options):
     source_role_arn = options.get('source_role_arn')
     destination_role_name = options.get('destination_role_name')
     external_id = options.get('external_id')
-    accounts_identifier = options.get('accounts_identifier')
+    accounts_connection_name = options.get('accounts_connection_name')
+
+    #Want to prepend 'data.' if it doesn't exist in the input
+    if accounts_connection_name[0:5] != 'data.':
```

Comment:
quick note; this will result in the input `DATA.AWS_ACCOUNTS_DEFAULT_CONNECTION` turning into `data.DATA.AWS_ACCOUNTS_DEFAULT_CONNECTION`; you'll want to make sure you know what case the string is (which can be done by calling .lower() on it, I think) before you try to do a comparison.

also, you can use `accounts_connection_name.startswith('data.')` as an alternative to slicing the string, if you like.